netmanager/src/redux/MainAlert/operations.js

```diff
@@ -18,7 +18,7 @@ export const updateMainAlert = (newAlertData) => {
         type: HIDE_ALERT_SUCCESS,
         payload: { ...state, show: false, timeout: null },
       });
-    }, 20000);
+    }, 3000);
```

Comment:
@ja-odur, how about if I set the delay to 8seconds when the alert severity = 'error' and the rest to 5 seconds?
`newAlertData.severity === 'error'? 8000: 5000`

Reply:
Yes @123MwanjeMike. Maybe we could explore using 10 and 5 seconds respectively.

Reply:
Thanks @ja-odur. Let me make the update and push asap

Reply:
Hi @123MwanjeMike , @ja-odur , thanks for these changes. 10 and 5 seconds sounds good. 

Just to ensure that we completely close this issue/request, I suggest that we explore going lower to even about 3 seconds for the rest. The user (Joel) wishes the success alerts disappear almost immediately.

Reply:
Thanks @Baalmart.
Let me make an update to reflect this.